src/backlight/portfolio/portfolio.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+import numpy as np
+from typing import List
+
+from backlight.positions.positions import Positions
+from backlight.datasource.marketdata import MarketData
+from backlight.metrics.position_metrics import calc_pl
+
+
+class Portfolio:
+    """
+    An abstract definition of a Portfolio is defined as a list of Positions
+    Each element of the list represent the positions of an asset that contribute to the portfolio
+    """
+
+    def __init__(self, positions: List[Positions]):
+        self.portfolio = positions
+        self.pl = pd.DataFrame()
+
+    @property
+    def value(self) -> pd.DataFrame:
```

Comment:
Why do you decide to add property of `value`, or `pl`?

When you define methods for the class, you need to think about following things:
1. What methods should the class have? Are they enough for every usecase? (at least, what we can imagine now)
2. Are there no duplicated methods in the class? e.g. If you can calculate `pl` with the `amount` of each asset, the class need not to have a method for `pl`.

Reply:
If we think about the portfolio as list of positions on each asset, I think we have to have a method that compute the total value of the portfolio at each time step. 
Let us decide/discuss what methods are needed..